.github/workflows/smoke_test.yml

```diff
@@ -1,6 +1,8 @@
 name: Smoke Test
 
 on:
+  schedule:
+    - cron: '*/5 * * * *'
```

Comment:
What's this scheduling?

Reply:
Copy paste artifact, I think for time being we don't need it.